src/stylesheets/components/_tag.scss

```diff
@@ -39,7 +39,7 @@
   }
 }
 
-@media only screen and (max-width: 600px) {
+@media only screen and (max-width: 640px) {
```

Comment:
if you use utility classes it will make this component much more flexible - example `sam-styles/src/components/patterns/page/page-with-sidebar.njk` `line 54, 65, 76`

Reply:
The designers asked that this never be shown on mobile versions, and it behaves odd if I don't make it display:none at 640px width